src/main/resources/templates/courses/assignments/group-delivery-review.ftl

```diff
@@ -170,6 +170,20 @@
 	$(computeTotals)
 	$('[type="radio"]').on('change', computeTotals)
     $('[type="radio"]').on('change', persistMasteries)
+
+	var down = "octicon-chevron-down"
+	var right ="octicon-chevron-right"
+
+	$('[data-target*="collapse-target"]').click(function() {
```

Comment:
I think this query-selector is rather expensive. It might be better to instead add a class `collapse-target` and do `document.querySelector('.collapse-target')` and then put the actual value in `data-target="${task?index}"`. You can then retrieve the index with `tableRow.dataset.target`